src/components/profile-view/favorite-movies.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Col, Row, Figure, Button, Card } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+
+
+function FavoriteMoviesComponent({ favoriteMovieList, removeFavoriteMovie }) {
+
+  const removeFav = (movieId) => {
+    removeFavoriteMovie(movieId);
+  };
```

Comment:
This function is a little redundant because you use the prop: `removeFavoriteMovie` directly without having to wrap it in another function.